components/meter/meter-linear.js

```diff
@@ -1,29 +1,46 @@
 import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { bodySmallStyles } from '../typography/styles.js';
 
 class MeterLinear extends LitElement  {
 	static get properties() {
 		return {
-			value: { type: Number },
+			context: { type: String },
 			max: { type: Number },
-			primary: { type: String },
-			secondary: { type: String }
+			percent: { type: Boolean },
+			singleLine: { type: Boolean, attribute: 'single-line' },
+			value: { type: Number }
 		};
 	}
 	static get styles() {
-		return [ css`
+		return [
+			bodySmallStyles,
+			css`
 			:host {
 				display: block;
 				position: relative;
 			}
 
+			.d2l-meter-linear-block {
```

Comment:
Perhaps `.d2l-meter-linear-block` and `.d2l-meter-linear-flex` can be replaced by:
```
:host > div {
	display: block;
}

:host([single-line]) > div {
	align-items: center;
	display: flex;
	flex-direction: row;
}
```
I think it would avoid the class logic further below in the `render()`.  I think it may also made these selectors easier to reason about.

Reply:
Right! I knew I was going about it in a weird way.